content/en/synthetics/_index.md

```diff
@@ -8,73 +8,49 @@ further_reading:
 - link: "https://www.datadoghq.com/blog/introducing-synthetic-monitoring/"
   tag: "Blog"
   text: "Introducing Datadog Synthetics"
-- link: "/synthetics/api_tests/"
+- link: "/synthetics/troubleshooting/"
   tag: "Documentation"
-  text: "Configure an API Test"
-- link: "/synthetics/browser_tests/"
+  text: "Troubleshooting Synthetics"
+- link: "/synthetics/settings/"
   tag: "Documentation"
-  text: "Configure a Browser Test"
-- link: "/synthetics/identify_synthetics_bots/"
-  tag: "Documentation"
-  text: "Identify Synthetics Bots"
+  text: "Synthetics Settings"
 ---
 
-## Overview
-
-Datadog Synthetics gives you a new layer of visibility on the Datadog platform. By monitoring your applications and API endpoints via simulated user requests and browser rendering, Synthetics helps you ensure uptime, identify regional issues, and track application performance. By unifying Synthetics with your metrics, traces, and logs, Datadog allows you to observe how all your systems are performing as experienced by your users.
-
 {{< img src="synthetics/synthetics_home_page.png" alt="Synthetics home page" >}}
 
-## Search
-
-Advanced search lets you query checks by any combination of checks attributes:
-
-* `title` and `message` - text search
-* `status` - Alert, No Data, Ok
-* `creator`
-* `region`
-* `muted`
-* `notification`
-* `tags`
+Synthetic tests allow you to observe how all your systems are performing as experienced by your users from around the globe -  Datadog tracks the performance of your applications and API endpoints using simulated user requests and browser rendering. Synthetics helps you ensure uptime, identify regional issues, compute error budgets, automate tests, and track application performance. By unifying Synthetics with your metrics, traces, and logs, you can test your production environment end-to-end.
```

Comment:
```suggestion
Synthetic tests allow you to observe how all your systems are performing using simulated user requests from around the globe. Datadog tracks the performance of your webpages and APIs at all network levels (HTTP, TCP, SSL) in a controlled and stable way, alerting you in case of faulty behavior (high response time, unexpected status code, etc.). Computing SLOs on your key endpoints and user journeys makes it easier to stick to your application performance targets and ultimately provide a consistent customer experience. By end-to-end testing both your production and CI environments, you can increase your development teams’ speed and stop fearing defective code might make it to production.
```